roles/openshift_node/tasks/main.yml

```diff
@@ -23,7 +23,7 @@
   - role: node
     local_facts:
       annotations: "{{ openshift_node_annotations | default(none) }}"
-      debug_level: "{{ openshift_node_debug_level | default(openshift.common.debug_level) }}"
+      debug_level: "{{ lookup('oo_option', 'openshift_node_debug_level') | default(openshift.common.debug_level, True) }}"
```

Comment:
Same comments as above about openshift_master_debug level. I don't want to extend the cli/env lookups into the roles if at all possible.